selfdrive/car/chrysler/carstate.py

```diff
@@ -3,150 +3,211 @@
 from opendbc.can.parser import CANParser
 from opendbc.can.can_define import CANDefine
 from selfdrive.car.interfaces import CarStateBase
-from selfdrive.car.chrysler.values import DBC, STEER_THRESHOLD
+from selfdrive.car.chrysler.values import DBC, STEER_THRESHOLD, RAM_CARS
 
 
 class CarState(CarStateBase):
   def __init__(self, CP):
     super().__init__(CP)
     can_define = CANDefine(DBC[CP.carFingerprint]["pt"])
-    self.shifter_values = can_define.dv["GEAR"]["PRNDL"]
+    self.shifter_values = can_define.dv["Transmission_Status"]["Gear_State"]
```

Comment:
We'll want to find a message that includes low, or maybe it's a bit in this message.